docs/changes.md

```diff
@@ -1,6 +1,7 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `12.15` Add additional AWS storage support for MVI playbook ([#712](https://github.com/ibm-mas/ansible-devops/pull/712))
```

Comment:
Can you also update change logs in `ibm/mas_devops/README.md` ?

Reply:
updated change log.